lib/vm.py

```diff
@@ -346,6 +346,10 @@ def detect_package_manager(self):
 
     def insert_cd(self, vdi_name):
         logging.info("Insert CD %r in VM %s", vdi_name, self.uuid)
+        # TODO: rescan only the right ISO
```

Comment:
Using "FIXME" instead makes it easier to find.
`insert_cd` may not be a good place, e.g. we use have some day some ISO caching.
It would be better to have some `copy_to_iso_sr` helper, which would take care of this sync.